frontend/packages/kubevirt-plugin/src/components/vms/vm.tsx

```diff
@@ -279,157 +260,69 @@ const VirtualMachinesPage: React.FC<VirtualMachinesPageProps> = (props) => {
-      // order of arrays designates the priority
-      loadedVMs,
-      loadedVMIs,
-      uniqueVMImportsByTargetName,
-      (entity: VMKind | VMIKind | VMImportKind) =>
-        entity.kind === VirtualMachineImportModel.kind
-          ? `${getNamespace(entity)}-${new VMImportWrappper(entity).getResolvedVMTargetName()}`
-          : getBasicID(entity),
-    );
-
-    return virtualMachines
-      .map((obj: VMILikeEntityKind | VMImportKind) => {
-        const lookupID = getBasicID(obj);
-        const objectBundle: ObjectBundle = { vm: null, vmi: null, vmImport: null };
-        let vmStatusBundle: VMStatusBundle;
-        let vmImportStatus: V2VVMImportStatus;
-
-        if (isVMImport(obj)) {
-          objectBundle.vmImport = obj;
-          const { vmImportStatus: importstatus, ...bundle } = getVMImportStatusAsVMStatus({
-            vmImport: obj,
-          });
-          vmStatusBundle = bundle;
-          vmImportStatus = importstatus;
-        } else {
-          if (isVM(obj)) {
-            objectBundle.vm = obj;
-            objectBundle.vmi = vmisLookup[lookupID];
-          } else if (isVMI(obj)) {
-            objectBundle.vmi = obj;
-          }
-
-          vmStatusBundle = getVMStatus({
-            vm: objectBundle.vm,
-            vmi: objectBundle.vmi,
-            pods: loadedPods,
-            migrations: loadedMigrations,
-            pvcs: loadedPVCs,
-            dataVolumes: loadedDataVolumes,
-            vmImports: loadedVMImports,
-          });
-        }
-
-        return {
-          metadata: {
-            name: getName(obj),
-            namespace: getNamespace(obj),
-            node: getVMINodeName(objectBundle.vmi),
-            creationTimestamp: getCreationTimestamp(obj),
-            uid: getUID(obj),
-            status: vmStatusBundle.status.toSimpleSortString(),
-            vmStatusBundle,
-            vmImportStatus,
-            lookupID,
-            labels: getLabels(obj),
-          },
-          ...objectBundle,
-        };
-      })
-      .filter(({ vmImport, metadata }) => !(vmImport && metadata.vmImportStatus?.isCompleted()));
+    ];
+
+    return virtualMachines.map((obj: VMILikeEntityKind) => {
+      const objectBundle: ObjectBundle = { vm: null, vmi: null, vmImport: null };
+      const lookupID = getBasicID(obj);
+
+      if (isVM(obj)) {
+        objectBundle.vm = obj;
+        objectBundle.vmi = vmisLookup[lookupID];
+      } else if (isVMI(obj)) {
+        objectBundle.vmi = obj;
+      }
+      return {
+        metadata: {
+          name: getName(obj),
+          namespace: getNamespace(obj),
+          creationTimestamp: getCreationTimestamp(obj),
+          node: getVMINodeName(objectBundle.vmi),
+          uid: getUID(obj),
+          lookupID,
+          labels: getLabels(obj),
+          // printableStatus is only available for latest kubevirt version, so if its not available we sort by ready
+          status: obj?.status?.printableStatus || obj?.status?.ready,
```

Comment:
where is the fallback to vm-status method when  obj?.status?.printableStatus is not available

Reply:
/hold
`obj?.status?.ready` is not the currect fallbak is should be` vmStatusBundle.status.toSimpleSortString()`